MonoGame.Framework.Content.Pipeline/Serialization/Intermediate/IntermediateWriter.cs

```diff
@@ -24,37 +34,180 @@ internal IntermediateWriter(IntermediateSerializer serializer, XmlWriter xmlWrit
 
         public void WriteExternalReference<T>(ExternalReference<T> value)
         {
-            throw new NotImplementedException();
+            var externalReference = new ExternalReference
+            {
+                ID = "#External" + (_externalReferences.Count + 1),
+                TargetType = typeof(T).FullName,
+                FileName = MakeRelativePath(value.Filename)
+            };
+            _externalReferences.Add(externalReference);
```

Comment:
Good point - I wondered about it too, but forgot to write a test for it. I'll fix that.